src/pages/news/news-page/news-page.js

```diff
@@ -19,17 +30,30 @@ const tableTitles = config.tableHeadRowTitles.news;
 
 const NewsPage = () => {
   const classes = useStyles();
-  const list = useSelector(({ News }) => News.list);
+  const { list, loading } = useSelector(({ News }) => ({
+    list: News.list,
+    loading: News.loading
+  }));
   const dispatch = useDispatch();
 
   useEffect(() => {
     dispatch(getNews());
   }, [dispatch]);
 
-  const loading = useSelector(({ News }) => News.loading);
+  const openSuccessSnackbar = (eventHandler) => {
+    dispatch(setDialogTitle(REMOVE_TITLE));
```

Comment:
Please, try to reduce this long list of action creators to two:
- `showDialog`that will take object with all of the data for the dialog
- `closeDialog` that will reset all of the data and close the dialog